.github/workflows/aws_tfhe_backward_compat_tests.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           echo "PULL_REQUEST_MD_LINK=[pull-request](${{ vars.PR_BASE_URL }}${{ github.event.pull_request.number }}), "  >> "${GITHUB_ENV}"
 
       - name: Slack Notification
-        if: ${{ failure() }}
+        if: ${{ failure() || cancelled() }}
```

Comment:
I'm going to be annoying here but I think we removed cancelled on tests to avoid spamming the channel for notifications

Reply:
could be good to keep on the long run tests however

Reply:
you could make it "generic" in the sense that it checks for the github event and this way you can copy/paste, up to you, but yeah cancelled on PRs not interesting

Reply:
Done.